docs/include/conf/current.rst

```diff
@@ -15,12 +15,16 @@
 
 - :doc:`Write the Docs Day Australia </conf/au/2017/index>`, November 24, **Melbourne, Australia**
 
+Want to run your own one-day Write the Docs event? Check our :doc:`guide for starting an event </organizer-guide/confs/start/>` and `email us <mailto:conf@writethedocs.org>`_ if you have any questions! 
```

Comment:
I thought we're retiring the conf@ address in favor of support@ and city-specific? (on a related note: we should go over the website and update all the addresses.

Reply:
Aye, I just copied it from below -- it makes some sense for this case, but yea should standardize.